frontend/styles/global.styles.ts

```diff
@@ -5,35 +5,61 @@ export const GlobalStyles = createGlobalStyle`
     body{
         background: ${({ theme }) => theme.body};
         color: ${({ theme }) => theme.text};
-        font-family: 'Inter', sans-serif;
+        margin: 0;
+        border-top: ${({theme}) => `8px solid ${theme.border}`};
     }
 
-    h1{
```

Comment:
Not to be a stickler, but a bunch of these styles are either `p {` or `p{` can we make sure we got a space between N and {

Reply:
sure.